src/hevm/hevm.cabal

```diff
@@ -150,6 +150,8 @@ executable hevm
     -Wall -threaded -with-rtsopts=-N
   other-modules:
     Paths_hevm
+  if os(darwin)
+    ld-options: -Wl,-keep_dwarf_unwind
```

Comment:
Can you explain a bit more?

Reply:
Yes, how to summarise - I spent a bit of time chasing this around :-) The issue turned out to be with some broken c++ exception handling on darwin, triggered by the try/catch behaviour in ethjet-ff.cc. These options are link time optimisation flags which allow fallback to dwarf exception tables, overriding the default GHC -no_compact_unwind which seems to cause the broken behaviour.

Reply:
Wow ok.